src/js/plugin/external/PluginHelper.js

```diff
@@ -1,27 +0,0 @@
-import PluginDispatcher from "./PluginDispatcher";
```

Comment:
At the moment you could simply add them via the `add` methods as the  `PluginMountPoints` are shared behind the scenes. But I would move them to the SDK and rather not share them.
Adding `PluginMountPoints` via plugins is an edge case, and as we don't have a plugin dependency mechanism nor any plugin order building a plugin that uses mount points from a different plugin is very fragile and I wouldn't recommend doing so. If you feel that we need to support this we should solve all the plugin dependency problems first and then talk about an elegant way to share those.